# Devoir 1

### Par Olivier Malenfant-Thuot
Matricule: 1012818

## Question 1: Small exercise on probabilities

In [3]:
cancer_proportion = 0.015
positive_if_healthy = 0.096
positive_if_sick = 0.87

total_positive = ( cancer_proportion * positive_if_sick + 
                  (1 - cancer_proportion) * positive_if_healthy )
sick_positive = ( cancer_proportion * positive_if_sick)

final_prob = sick_positive / total_positive
print(final_prob)

0.12127125731809309


La probabilité qu'une femme ait vraiment le cancer, même si son test est positif, est de seulement 12 %.

La bonne réponse est E).

## Question 2: Curse of dimensionality and geometric intuition in higher dimensions